common/test/java/com/couchbase/lite/LiveQueryTest.java

```diff
@@ -116,10 +119,7 @@ public void testLiveQueryDelay() throws CouchbaseLiteException, InterruptedExcep
             .where(Expression.property(KEY).greaterThanOrEqualTo(Expression.intValue(0)))
             .orderBy(Ordering.property(KEY).ascending());
 
-        // There should be two callbacks:
-        //  - immediately on registration
-        //  - after LiveQuery.UPDATE_INTERVAL_MS when the change gets noticed.
-        final long[] times = new long[] {1, System.currentTimeMillis(), 0, 0};
+        final long[] times = new long[] {1, System.currentTimeMillis(), 0};
```

Comment:
so... why *remove* documentation?  I can see fixing it, if it is wrong.  Removing it, tho?  rrrrrr..